system/Database/BaseBuilder.php

```diff
@@ -2625,18 +2638,8 @@ protected function compileWhereHaving($qb_key)
 						continue;
 					}
 
-					// $matches = array(
```

Comment:
Put these back in please. They serve as a nice reference when working on that code...